app/src/main/java/org/jellyfin/mobile/fragment/WebViewFragment.kt

```diff
@@ -129,15 +130,20 @@ class WebViewFragment : Fragment() {
                 val url = request.url
                 val path = url.path?.toLowerCase(Locale.ROOT) ?: return null
                 return when {
-                    path.endsWith(Constants.APPLOADER_PATH) -> {
+                    path.endsWith(Constants.APPLOADER_PATH) || path.endsWith(Constants.MAIN_BUNDLE_PATH) -> {
+                        if (path.endsWith(Constants.APPLOADER_PATH))
+                            assetsVersion = "10.6"
+                        else if (path.endsWith(Constants.MAIN_BUNDLE_PATH))
+                            assetsVersion = "10.7"
```

Comment:
Could make this a `when` as well, and have the else case throw an error.

Reply:
Don't need an else clause because there's only 2 possible branches here considering we matched with one of both paths in the parent when branch